mito-ai/src/Extensions/AiChat/ChatHistoryManager.tsx

```diff
@@ -242,32 +248,10 @@ export class ChatHistoryManager {
 const getDisplayedOptimizedUserMessage = (input: string, activeCellCode?: string): OpenAI.Chat.ChatCompletionMessageParam => {
     return {
         role: 'user',
-        content: `\`\`\`python
+        content: activeCellCode ? `\`\`\`python
 ${activeCellCode}
 \`\`\`
 
-${input}`};
-}
-
-const removeInnerThoughtsFromMessage = (messageContent: string) => {
```

Comment:
I think we're going to need this sort of post-processing code in the frontend for when we stream responses to the frontend.